drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -127,7 +143,7 @@ class DragAndDropBlock(XBlock, XBlockWithSettingsMixin, ThemableXBlockMixin):
         default={},
     )
 
-    num_attempts = Integer(
+    attempts = Integer(
```

Comment:
Correct - this field was introduced in the previous PR, #86, and has never been released.